local_ci.sh

```diff
@@ -30,6 +32,10 @@ while [[ $# -gt 0 ]]; do
     TEST=1
     shift
     ;;
+  -u | --updateSessionReplayPayload)
```

Comment:
I wouldn't use `-u` for that and keep only long version of argument. `-u` is not quite self-descriptive and this task anyway is specific for the single module.

Reply:
yeah makes more sense